CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## [Unreleased]
 ### Fixed
 - Fix style regression in DatePickerSingle and DatePickerRange [#518](https://github.com/plotly/dash-core-components/issues/518)
+- In `dcc.Input`, fixed several HTML properties that weren't properly camel cased and therefore were not actually applied in the DOM [#523](https://github.com/plotly/dash-core-components/pull/523/):
+    - `autocomplete` is now `autoComplete`
+    - `autofocus` is now `autoFocus`
+    - `inputmode` is now `inputMode`
+    - `maxlength` is now `maxLength`
+    - `minlength` is now `minLength`
+- Updated property definitions of various components in anticipation of upcoming component validation (https://github.com/plotly/dash-renderer/pull/100/). [#523](https://github.com/plotly/dash-core-components/pull/523/)
```

Comment:
updated, or fixed? Perhaps "improved" since it's a little of each I guess.

Reply:
:+1: for improved, or we could have two categories in release log: 1 bug fix 2 feature enhancement